{"frames":[],"message":"Error in covid.cases.cumcases3(X = seq(1:168), Y = cat$num_casos, max.cases.free = T,  : \n  no se pudo encontrar la funci√≥n \"covid.cases.cumcases3\"\n"}